libexec/alluxio-config.sh

```diff
@@ -90,33 +90,57 @@ ALLUXIO_SERVER_CLASSPATH="${ALLUXIO_CONF_DIR}/:${ALLUXIO_CLASSPATH}:${ALLUXIO_AS
 ####################################################################################################
 function getConf {
   "${JAVA}" -cp ${ALLUXIO_CLIENT_CLASSPATH} ${ALLUXIO_JAVA_OPTS} -Dalluxio.logger.type=Null \
+      -Dalluxio.remote.logger.type=Null \
```

Comment:
Another alternative is to add the following line in log4j.properties
log4j.appender.=org.apache.log4j.varia.NullAppender